.claude/skills/uts-to-swift/SKILL.md

```
@@ -477,82 +664,556 @@ Helper methods return types should be ready for use in the test code without add
+swift test --filter "UTS.<className>/<test_method_name>"
 ```
 
-(`--filter` takes a regex over test names; `swift test --filter UTS` runs the whole suite.)
+(`swift test --filter UTS` still runs the whole UTS target — including the integration tiers. There is
+no env-var gating: integration and proxy suites run directly from `swift test` or the Xcode test runner;
+they just need network access, and the proxy tier needs macOS — see `Test/UTS/README.md` §10.)
 
-Handle failures using this decision tree (see [reference doc](https://github.com/ably/specification/blob/main/uts/docs/writing-derived-tests.md)):
+Handle test failures using this decision tree (the **Required reading** doc you fetched up front has the full detail):
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)